elastalert/ruletypes.py

```diff
@@ -191,6 +204,7 @@ def add_match(self, match):
         if change:
             extra = {'old_value': change[0],
                      'new_value': change[1]}
+            elastalert_logger.info("Matching things " + str(dict(match.items() + extra.items())))
```

Comment:
Add more context to the log message. `Matching things` could mean so many things inside elastalert.